src/sentry/api/endpoints/group_details.py

```diff
@@ -199,9 +218,13 @@ def get(self, request, group):
 
         action_list = self._get_actions(request, group)
 
-        if first_release:
+        if first_release is not None and last_release is not None:
```

Comment:
We make snuba calls to get `first_release` and `last_release` as well. Are those calls to snuba also slow?

Reply:
:+1: They aren't, only take about 1/20th the time to run. Could investigate doing 1 big query instead to get first/last seen for both first/last release, but not worth the effort here I think 🤔